src/components/Confirm/style.css

```diff
@@ -18,9 +18,16 @@
   padding: 0;
 }
 
-.fileViewer{
-  width: 92%;
-  height: 290px;
-  max-height: 70vh;
-  border: 0;
+:global {
```

Comment:
You should always question what's going on if you're making `:global` stuff for your own code (rather than just for when integrating with third party CSS where it's unavoidable)

In this case, anyway - I believe you don't need to the `:global`, since IDs are just inherently global

But yeah, as per my other comment - this could just be a class of the `a`, I think?